### DISCUSSION TITLE
Hide Communities from All, Help guide

### DISCUSSION THREAD

**Discussion by oohgodyeah** [aeharding/voyager#300] [Q&A] [answered]
How do you hide entire communities/channels/magazines from All or Local?

Is there a help guide/documentation anywhere?

**Comment by adjoseph16** [comment, marked as answer]
This isn’t implemented within the app yet. You need to do it through the web UI. Here’s how it works on mobile (the process is very similar on desktop):


[Video of unsubscribe process](https://github.com/aeharding/wefwef/assets/78319881/a5321348-05ab-48b6-89ce-5b1388686b3b)

**Reply by oohgodyeah** [reply]
That's the answer I needed, thanks!
